db/migrate/20160505133601_rename_alternative_url.rb

```diff
@@ -0,0 +1,5 @@
+class RenameAlternativeUrl < ActiveRecord::Migration
+  def change
+    rename_column :unpublishings, :alternative_url, :alternative_path
```

Comment:
When you unpublish something, can you set an external url to redirect to? If so, would we need the full URL rather than just the path?